onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -159,6 +159,9 @@ void Check<MLFloat16>(const OpTester::Data& expected_data, const Tensor& output_
   }
 
   float threshold = 0.001f;
+#ifdef USE_TENSORRT
+  threshold = 0.005f;
```

Comment:
why does it need to be increased?

Reply:
is it because FP16 is now default?

Reply:
CUDA doesn't need this?

Reply:
It's originally for FP16 because of precision degradation.